jest.setup.js

```diff
@@ -5,3 +5,8 @@ require('jest-axe/extend-expect');
 require('@testing-library/jest-dom/extend-expect');
 
 require('jest-styled-components');
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useLayoutEffect: jest.requireActual('react').useEffect,
+}));
```

Comment:
Would this be good place for those rare comments? To reason why this is done.

Reply:
Let's see if I can come up with reasonable comments without getting too much into details.

Reply:
There should be some kind of description now. 😅

Reply:
I think those comments are good addition to clear things up 👍